doc/classes/HTTPRequest.xml

```diff
@@ -275,6 +275,7 @@
 			Request successful.
 		</constant>
 		<constant name="RESULT_CHUNKED_BODY_SIZE_MISMATCH" value="1" enum="Result">
+			Request failed due to a mismatch between the expected and actual chunked body size during transfer.
```

Comment:
This description needs some more details I think, right now it is just a different way to say what the name of the error says

Reply:
Ok, now I have added some possible causes too.